XML/Atom/Entry.php

```diff
@@ -17,98 +17,12 @@
  * A class used to generate an entry node.
  *
  * @package   XML_Atom
- * @copyright 2008-2016 silverorange
+ * @copyright 2008-2017 silverorange
  * @license   http://www.gnu.org/copyleft/lesser.html LGPL License 2.1
+ * @link      https://github.com/silverorange/xml-atom
  */
 class XML_Atom_Entry extends XML_Atom_Element
 {
-    // {{{ protected properties
```

Comment:
Should all these protected properties be deleted? I looked in the parents of this class and didn't see these variables listed anywhere.